README.md

```diff
@@ -106,3 +106,7 @@ It is easy to create a stateless API based on this stateful one, or vice versa,
 Q: Why is this an Intl API instead of String methods?
 
 A: All of these break types are actually locale-dependent, and some allow complex options. The result of the `segment` method is a SegmentIterator. For many non-trivial cases like this, analogous APIs are put in ECMA-402's Intl object. This allows for the work that happens on each instantiation to be shared, improving performance. We could make a convenience method on String as a follow-on proposal.
+
+Q: What exactly does the index refer to?
+
+An index *n* refers to the boundary preceding the code unit *n*. For example, when iterating over the string "Hello, world💙" by words in English, there will be boundaries at 0, 5, 6, 7, 12, and 1. The definition of these boundary indices does not depend on whether forwards or backwards iteration is used.
```

Comment:
```suggestion
An index *n* refers to the boundary preceding the code unit *n*. For example, when iterating over the string "Hello, world💙" by words in English, there will be boundaries at 0, 5, 6, 7, 12, and 13. The definition of these boundary indices does not depend on whether forwards or backwards iteration is used.
```

Reply:
This should actually say 14 (I used a blue, not red, heart to draw out the difference between code point and code unit)